apps/charterafrica/src/lib/openAfrica.js

```diff
@@ -23,62 +55,73 @@ async function formatDatasets(data) {
       type,
     } = dataset;
 
-    const formattedResources = resources?.map((resource) => {
-      const { url, format, name: resourceName, description } = resource;
-      return {
-        description,
-        format,
-        name: resourceName,
-        url,
-      };
-    });
+    const formattedResources = formatResources(resources, author);
 
-    const allDocumentFormats = [
+    const formats = [
       ...new Set(formattedResources?.map((resource) => resource.format)),
     ];
```

Comment:
```suggestion
    const formatsSet = new Set(resources?.map((r) => r.format));
    const formats = Array.from(formatsSet);
```

Shorter code is always easier to follow.